### PR TITLE
Flatten Contracts

### PR DESCRIPTION
**Pull request by itzmeanjan**
This PR flattens contract(s), which are to be used for deployment.

### CODE REVIEW

**Review by QEDK — CHANGES_REQUESTED** [review]
Can we unstage all files except `MintableERC1155Predicate.sol`? That is the only file with meaningful changes. And the truffle flattener is adding empty lines on empty lines... I am not sure why but it's not ideal.

**Comment by itzmeanjan** [comment]
> Can we unstage all files except `MintableERC1155Predicate.sol`? That is the only file with meaningful changes. And the truffle flattener is adding empty lines on empty lines... I am not sure why but it's not ideal.

I see.

File: flat/MintableERC1155Predicate.sol [hunk 1]
```diff
@@ -1363,6 +1371,9 @@ contract Initializable {
 pragma solidity 0.6.6;
 
 
+    ERC1155Receiver
```
Comment (QEDK) [inline flat/MintableERC1155Predicate.sol:1374]: Something went awry here it seems?

**Review by QEDK — CHANGES_REQUESTED** [review]

File: flat/MintableERC1155Predicate.sol (hunk 1 above)
Comment (itzmeanjan) [reply flat/MintableERC1155Predicate.sol:1374]: Truffle flattener issue.

**Review by QEDK — APPROVED** [review]
LGTM!

**Pull request merged**